source/reference/verification.txt

```diff
@@ -118,6 +118,41 @@ source to the destination cluster.
        :method:`db.getCollectionInfos` method on the source and
        destination clusters and compare the results.
 
+   * - .. _c2c-shardkey-comparison: 
+
+       Shard Key Comparison
+     - To verify the transfer of shard keys to a synced collection, run a query on the ``config.collections``
+       collection to find a document whose ``_id`` value is the namespace of the target collection. 
+       Compare the ``key`` value of this document in the source and destination clusters.
```

Comment:
[question] Should key be monospaced here?

Reply:
I am using "key" to refer to the field name in the document, so if ``_id`` is monospaced, it feels more consistent to also monospace ``key``. Do you think it would be helpful to emphasize that``key`` is a field name? eg 

> "Compare the ``key`` field value..."